jdate-class.js

```diff
@@ -133,9 +133,9 @@ JDate.prototype = {
 		var persian = this._persianDate();
 		persian[which] = value;
 		var new_date = jd_to_gregorian(persian_to_jd_fixed(persian[0], persian[1], persian[2]));
-		this._d.setFullYear(new_date[0]);
-		this._d.setMonth(new_date[1] - 1);
 		this._d.setDate(new_date[2]);
+		this._d.setMonth(new_date[1] - 1);
+		this._d.setFullYear(new_date[0]);
```

Comment:
Thanks for contribution.
Can you explain why you have changed the order of these statements? Normally date should be set after month, not before it.
This effectively breaks the date conversion (jsfiddle for your pull request: http://jsfiddle.net/085v844g/)